readthedocs/oauth/notifications.py

```diff
@@ -30,3 +32,23 @@ def get_context_data(self):
             'url_docs_webhook': 'http://docs.readthedocs.io/en/latest/webhooks.html',  # noqa
         })
         return context
+
+
+class InvalidProjectWebhookNotification(SiteNotification):
+
+    context_object_name = 'project'
+    failure_level = ERROR_PERSISTENT
+    failure_message = _(
+        "You project {{ project.name }} doesn't have a valid webhook set up, "
```

Comment:
`You` -> `The` or `Your` -- I prefer to not infer ownership, as this doesn't translate to a commercial organization controlling the project.